### PR TITLE
[test] add hubble system test to CI

### PR DESCRIPTION
**Pull request by matmerr**
**Reason for Change**:

Validate existing Cilium scenarios with Hubble enabled

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

File: test/integration/manifests/cilium/cilium-config.yaml [hunk 1]
```diff
@@ -42,6 +42,16 @@ data:
   enable-vtep: "false"
   enable-well-known-identities: "false"
   enable-xt-socket-fallback: "true"
+  hubble-metrics: flow:sourceContext=pod-short;destinationContext=pod-short
+    tcp:sourceContext=pod-short;destinationContext=pod-short
+    dns:flow:sourceContext=pod-short;destinationContext=pod-short
+  hubble-metrics-server: :9965
+  hubble-disable-tls: "false"
```
Comment (matmerr) [inline test/integration/manifests/cilium/cilium-config.yaml:49]: maybe?

File: test/integration/manifests/cilium/cilium-config.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ data:
   enable-health-check-nodeport: "true"
   enable-health-checking: "true"
   enable-host-legacy-routing: "true"
-  enable-hubble: "false"
+  enable-hubble: "true"
```
Comment (vakalapa) [inline test/integration/manifests/cilium/cilium-config.yaml:27]: lets create a separate config map ?? and keep this as is

File: test/integration/manifests/cilium/cilium-config-hubble.yaml [hunk 3]
```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
```
Comment (tamilmani1989) [inline test/integration/manifests/cilium/cilium-config-hubble.yaml:1]: where you are deploying this cilium config? I didn't see this config getting deployed in cilium-e2e-step-template

File: test/integration/manifests/cilium/cilium-config-hubble.yaml [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+data:
```
Comment (tamilmani1989) [inline test/integration/manifests/cilium/cilium-config-hubble.yaml:2]: what flags did you change from the one without hubble?

**Comment by tamilmani1989** [comment]
@matmerr instead of adding this in PR pipeline we should add it in nightly pipeline.. PR pipeline is mainly to catch regressions with cns and azure-ipam component..cilium image would be always same here

**Comment by matmerr** [comment]
cc: @jshr-w

**Pull request merged**